src/log/log_slot.c

```diff
@@ -526,7 +526,7 @@ __wt_log_slot_join(WT_SESSION_IMPL *session, uint64_t mysize, uint32_t flags, WT
     log = conn->log;
     time_start = time_stop = 0;
 
-    WT_ASSERT(session, !F_ISSET(session, WT_SESSION_LOCKED_SLOT));
+    WT_ASSERT(session, !FLD_ISSET(session->lock_flags, WT_SESSION_LOCKED_SLOT));
```

Comment:
Can we create a macro to check the lock flags? I feel it is accessed a lot.

Reply:
Expect for `flags` in the code we are currently referring to the `*_flags` directly wherever they are accessed. Adding a macro also should be specific to the session as the `lock_flags` is specific to the session. I don't find that it improves the code much.